CHANGELOG.md

```diff
@@ -7,3 +7,15 @@ This format is based on [Keep A Changelog](https://keepachangelog.com/en/1.0.0).
 ### Added
 
 * First release
+
+## 1.0.1 -- 2022-05-17
+
+### Added
+
+* `alwaysFailProperty` -- an universial property that ensures script always fails
+
+### Fixed
+
+* Bug that made `classifiedProperty` fail to differentiate between
+  wrong result and expected crash when script ran without
+  crashing. (Fixed #5)
```

Comment:
I believe the convention is that more recent changelogs go earlier in the list than older ones. Does KeepAChangelog specify otherwise? It's more convenient for readers.